fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java

```diff
@@ -1181,8 +1181,8 @@ public void subscribeToLao(String laoId) {
                 lao -> {
                   Log.d(TAG, "got an update for lao: " + lao.getName());
                   mCurrentLao.postValue(lao);
+                  assert keyManager != null;
```

Comment:
Isn't `keyManager` injected through Hilt ?
When is this assert needed ?

Reply:
I forgot to remove it sorry